File: client/src/app/interfaces/game.ts
import { Question } from '@app/interfaces/question';

export interface Game {
    id: string;
    originalId?: string;
    authorId: string;
    authorName: string;
    title: string;
    description: string;
    lastModification: string;
    duration: number;
    isVisible?: boolean;
    nMatchesPlayed?: number;
    questions: Question[];
}

File: client-mobile/app/src/main/java/com/example/polyquiz/Game.kt
package com.example.polyquiz

import com.example.polyquiz.match.domain.Question

data class Game(
    val id: String? ="",
    val originalId: String,
    val title: String,
    val description: String,
    val questions: List<Question>? = null,
    val lastModification: String?,
    val isVisible: Boolean?,
    val nMatchesPlayed: Number,
    val duration: Number,
)
